x-pack/plugins/fleet/common/constants/epm.ts

```diff
@@ -15,6 +15,7 @@ export const requiredPackages = {
   System: 'system',
   Endpoint: 'endpoint',
   ElasticAgent: 'elastic_agent',
+  SecurityDetectionEngine: 'security_detection_engine',
```

Comment:
What happens if this dependency is removed in the future -- is the package just gracefully uninstalled and SO's removed on upgrade?

Reply:
yeah you got it.
fleet manages all the SOs, so they will be removed during an upgrade/uninstall gracefully. the detection engine will then just use the rules compiled in

Reply:
maybe I misunderstood the question but if `security_detection_engine` is removed from this list of required packages in the future, Fleet will *not* automatically uninstall the package if it's already installed

Reply:
Ahhh, so we can have lingering SO's from this package then? This would result in the rules installing from these SO's still, or would the fact the package is no-longer present short circuit that logic?

How can they be manually removed -- will it still show up in fleet until it's un-installed, or is there another way to perform this cleanup?